adhocracy4/polls/static/PollDetail/PollQuestions.jsx

```diff
@@ -383,7 +383,7 @@ class PollQuestions extends React.Component {
         <>
           {this.state.showResults
             ? (
-              <div className="pollquestionlist-container">
+              <div className="poll__preliminary-results">
```

Comment:
just to check: there are no styles for this in a+, that's intentional? and by removing the old class we lose a a padding-top, that's also intentional?

Reply:
Thanks, I can see it in the mb prs but in the a+ one the new class hasn't been added anywhere: https://github.com/liqd/adhocracy-plus/pull/2848/files